lib/cache/collections.go

```diff
@@ -33,13 +33,11 @@ type collection interface {
 	// will apply said resources to the cache.  fetch *must*
 	// not mutate cache state outside of the apply function.
 	fetch(ctx context.Context) (apply func(ctx context.Context) error, err error)
-	// process processes event
+	// processEvent processes event
 	processEvent(ctx context.Context, e types.Event) error
 	// watchKind returns a watch
 	// required for this collection
 	watchKind() types.WatchKind
-	// erase erases all data in the collection
-	erase(ctx context.Context) error
```

Comment:
Was this unused?

Reply:
Yes, I think that not the only method that was not used. I didn't want to clean up all of them as this is not the goal of this PR.